src/components/pipeline-list/pipeline-list.scss

```diff
@@ -17,7 +17,12 @@
 
   .kui-dropdown__label,
   .kui-menu-option {
-    height: 70px;
+    height: 58px;
+  }
+
+  .kui-dropdown__label,
+  .kui-menu-option__content {
+    font-size: 1.8em;
```

Comment:
I completely agree, this is another one that came from @GabrielComymQB who asked me to change it to follow the design after I'd left it as it was. I'm inclined to remove this override completely and just use the default value from Kedro-UI, which was 1.6em (16px)

Reply:
Hi everyone

Of course. The fact is you are now just having the impression it looks big right because you had the idea of reducing the cell height from 70px to 58px. 

I'm fine with the change for now.

Confirm:
58px cell height and 16px label font size.

We have to talk about it later tho.
Random sudden changes affect not only the element itself bu the whole. As you can see below. I hope you can see the balance here. Both are great in my opinion. But we are wasting some time here every time we have to make decisions again. We have to get better at it. Thanks

Great work

![Screenshot 2020-10-27 at 11 41 00](https://user-images.githubusercontent.com/55446576/97296957-4ab53180-1849-11eb-9d1d-9f0252a963be.png)